packages/core/src/oidc/adapter.ts

```diff
@@ -23,13 +23,15 @@ export default function postgresAdapter(modelName: string): ReturnType<AdapterFa
       name: client_name,
       type,
       oidcClientMetadata,
+      customClientMetadata,
     }: ApplicationUpdate): AllClientMetadata => ({
       client_id,
       client_name,
       application_type: getApplicationTypeString(type),
       grant_types: ['authorization_code', 'refresh_token'],
       token_endpoint_auth_method: 'none',
       ...snakecaseKeys(oidcClientMetadata),
+      ...customClientMetadata, // OIDC Provider won't camelcase custom parameter keys
```

Comment:
so we snake_case this data?

Reply:
camelcase, align the usage as other OIDC client metadata which are consumed in camecase